server/http.go

```diff
@@ -915,3 +863,45 @@ func (p *Plugin) slackAttachmentToUpdatePMI(currentValue, channelID string) *mod
 
 	return &slackAttachment
 }
+
+func (p *Plugin) handleMeetingCreation(channelID, rootID, topic string, user *model.User, zoomUser *zoom.User) (string, error) {
+	var meetingID int
+	var createMeetingErr error
+	userPMISettingPref, err := p.getPMISettingData(user.Id)
+	if err != nil {
+		return "", errors.Wrap(err, "Error fetching PMI setting data")
```

Comment:
```suggestion
		return "", errors.Wrap(err, "error fetching PMI setting data")
```

See https://google.github.io/styleguide/go/decisions.html#error-strings